galpy/potential/planarForce.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, amp=1.0, ro=None, vo=None):
         self._amp = amp
         self.dim = 2
         self.isNonAxi = True
+        self.isDissipative = False
```

Comment:
Would it be sensible to set this in `planarPotential` instead? Otherwise there are a few Potentials that don't pick up an `isDissipative` attribute.

I.e., 
```
galpy/potential/WrapperPotential.py:class planarWrapperPotential(planarPotential):
galpy/potential/HenonHeilesPotential.py:class HenonHeilesPotential(planarPotential):
galpy/potential/CosmphiDiskPotential.py:class CosmphiDiskPotential(planarPotential):
galpy/potential/planarPotential.py:class planarAxiPotential(planarPotential):
galpy/potential/planarPotential.py:class planarPotentialFromFullPotential(planarPotential):
galpy/potential/SteadyLogSpiralPotential.py:class SteadyLogSpiralPotential(planarPotential):
galpy/potential/TransientLogSpiralPotential.py:class TransientLogSpiralPotential(planarPotential):
galpy/potential/EllipticalDiskPotential.py:class EllipticalDiskPotential(planarPotential):
```

Reply:
Yes, setting ``self.isDissipative = True`` in ``planarPotential`` would be the right thing to do, since everything that inherits from that class is non-dissipative.